### PR TITLE
Added MUI DatePicker and Fixed All Production Bugs

### PR DESCRIPTION
**Pull request by Lalit3716**
- Added Assignment Table
- Added Nested Routing on CoursePage
- Fixed Production Bugs
- Added MUI DateTimePicker

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+			router.events.off('routeChangeComplete', handleChangeComplete);
+		};
+	}, [router]);
+
 	return (
 		<Fragment>
 			<Head>
 				<title>Homework Scheduler</title>
 			</Head>
 			<AuthContextProvider>
```
[the remark this answers is not in the document]
Comment (Lalit3716) [reply pages/_app.js:35]: I tried this but it's somehow not working

**Comment by KShivendu** [comment]
@Lalit3716 @ambarvm Please wrap up this project asap.

**Comment by Lalit3716** [comment]
> @Lalit3716 @ambarvm Please wrap up this project asap.

It's complete you can just merge this Pull Request.

**Comment by Lalit3716** [comment]
@ambarvm Can we please merge this PR. We can work on some more features or performance optimizations afterwards ?

**Pull request merged**